feature-libs/product/configurators/common/src/components/config-previous-next-buttons/config-previous-next-buttons.component.ts

```diff
@@ -60,7 +58,7 @@ export class ConfigPreviousNextButtonsComponent implements OnInit {
       );
   }
 
-  scrollToVariantConfigurationHeader() {
+  protected scrollToVariantConfigurationHeader() {
     const theElement = document.querySelector('.VariantConfigurationTemplate');
```

Comment:
Can you help me understand what is this component doing?
And can you tell me how to see it in action?

Reply:
I can show you this in our session next week, but here is already the explanation:
The list of attributes can be quite long, so the user scrolls down and at the end there are the previous/next buttons. If he clicks on the buttons the application navigates to the previous/next group of attributes. Without the method and if the previous/next group is also quite long the user would end up somewhere in the middle or at the end of this group. Our UI designer decided that with every group change the UI should scroll to the start of the list of attributes so that the user is not missing attributes by accident.
Furthermore he decided that the UI should not scroll completely to the top but really to the start of the configuration area so that the user can concentrate on his work in the configuration session.

Reply:
Done. Disabled code execution for SSR.